app/presenters/hyrax/collection_presenter.rb

```diff
@@ -80,7 +81,7 @@ def total_viewable_collections
     end
 
     def collection_type_badge
-      collection_type.title
+      content_tag(:span, collection_type.title, class: "badge", style: "background-color: " + collection_type.badge_color + ";")
```

Comment:
Can you use `class: 'label'` instead of badge?  That makes it make the style used by the visibility markers.